docs/content/en/docs/concepts/metrics/_index.md

```diff
@@ -13,56 +13,63 @@ A `KeptnMetric` is a CRD representing a metric.
 The metric will be collected from the provider specified in the
 specs.provider.name field.
```

Comment:
```suggestion
`specs.provider.name` field.
```
Or is this the sort of formatting that is ill-advised here?  If so, just reject my suggestion ;-)